pkg/action/build-iso.go

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
I don't think we should bring the build parts too. There is another story to move those to osbuilder (https://github.com/kairos-io/kairos/issues/711). Until we do, I guess it's ok to keep them in the old "elemental-cli" binary, what do you think? Or are we planning to replace elemental-cli with kairos-agent first and then move the build parts to osbuilder?

Reply:
yep yep, this is a "fast, dirty" job to bring the libs in here, there is still lot of work to clean this up and unify, but I didnt want to block this further. Removing those parts can be done afterwards in parallel with the config merge and others.